### ISSUE TITLE
A11y_VSCodespaceBenchmark_Defaultsettingfornewcodespaces_DefaultSetting_ColorContrast : The "Standard (Linux) : 4 ..." content has color contrast ratio less than 4.5:1.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#106111]
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:​
Application Name: VS Codespace
Application Version: 1.48.0-insider
Commit: 0a7364f00514c46c9caceece15e1f82f82e3712f
Date: 2020-07-22T05:43:52.797Z
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating System: MAC 10.15.6
Screen Reader: Voiceover
Standard Violated: WCAG 1.4.3

### Repro Steps:​
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4. Navigate till "+" button beside the  "VS codespace" control and click on it .
5. Select any existing  Codespacesand then again click on the "+" button and navigate till "Default setting".
6. Using color contrast analyser check if the "Standard (Linux) : 4 ..." content has color contrast ratio more than or equal to 4.5:1.

### Actual:
The "Standard (Linux) : 4 ..." content has color contrast ratio of 3.0:1 which is less than the standard color contrast ratio of 4.5:1 which is not appropriate as the users with low vision will have difficulty in reading the text on the screen.

### Expected: ​
The color contrast ratio of "Standard (Linux) : 4 ..." content should be greater than or equal to 4.5:1.

### ​User Impact:
If the color contrast ratio of the content present on the screen is less than 4.5:1 then the users with low vision will have difficulty in reading the data on the screen.

Attachment:
<img width="1482" alt="1166134_A11y_VSCodespaceBenchmark_Defaultsettingfornewcodespaces_DefaultSetting_ColorContrast The  Standard (Linux) 4   content has color contrast ratio less than 4 51" src="https://user-images.githubusercontent.com/52036772/92232075-81b33a80-eecb-11ea-92cd-c5dbf2a5fce9.png">

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!

**Comment by MayuriNaik** [comment]
@isidorn We have already logged this bug for VS Codespace and they have redirected us to log an external bug for VS Code. So we logged this bug here.
